worker/src/main/java/org/apache/celeborn/service/deploy/worker/memory/MemoryManager.java

```diff
@@ -207,7 +203,7 @@ private MemoryManager(CelebornConf conf) {
         () ->
             logger.info(
                 "Direct memory usage: {}/{}, disk buffer size: {}, sort memory size: {}, read buffer size: {}",
-                Utils.bytesToString(nettyMemoryCounter.get()),
+                Utils.bytesToString(PlatformDependent.usedDirectMemory()),
```

Comment:
I think there needs a check to make sure that Celeborn Worker has been configured with off-heap memory. Or Celeborn will encounter undefined behavior.

Reply:
I think adding an assertion about  "PlatformDependent.usedDirectMemory() != -1 " could be enough.

Reply:
make sense, addressed.